<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStudWindow</class>
 <widget class="QDialog" name="addStudWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name            </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>English mark </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="engMark"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ukraine mark</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ukrMark"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Math Mark    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mathMark"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Enter">
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
